.storybook/preview.js

```diff
@@ -7,11 +7,10 @@ import theme from '../src/theme'
 export const parameters = {
 	actions: { argTypesRegex: '^on[A-Z].*' },
 	options: {
-		/** Sort alphabetically */
-		storySort: (a, b) =>
-			a[1].kind === b[1].kind
-				? 0
-				: a[1].id.localeCompare(b[1].id, undefined, { numeric: true })
+		storySort: {
+			method: 'alphabetical',
+			order: ['Foundation', 'Components', 'Demo']
+		}
```

Comment:
From my pov, there is important stuff in Foundation category and I believe that it should be visible on the first look. What the you think about moving it above Components category?

Reply:
Brilliant idea! Yes 😊 Thanks